cypher/cypher-docs/src/docs/dev/ql/constraints/syntax.adoc

```diff
@@ -21,6 +21,20 @@ It may still throw an error if conflicting data, indexes, or constraints exist.
 Examples of this are nodes with missing properties, indexes with the same name, or constraints with same schema but a different constraint type.
 
 For constraints that are backed by an index, the index provider and configuration for the backing index can be specified using the `OPTIONS` clause.
+Valid values for the index provider are `native-btree-1.0`, `lucene+native-3.0`, and `range-1.0`, default is `native-btree-1.0`.
```

Comment:
This is correct, but if planner gets done in time I guess we should mention that the first 2 are deprecated?

Reply:
Not sure if we mention it here or just in the deprecation section

Reply:
I think it might be nice to just write it as

Valid values for the index provider are `native-btree-1.0` (deprecated), `lucene+native-3.0` (deprecated), and `range-1.0`, default is `native-btree-1.0`.`

So that users get a hint that they should preferably use `range-1.0`